resources/assets/css/markdown-editor.css

```diff
@@ -1,4 +1,12 @@
+@import "_tailwind.css";
+
+/** UI Repo Styles **/
+@import "../../assets/css/styles-with-prism.css";
```

Comment:
why does it need to import tailwind and all the other styles? those will already be included in the project itself

Reply:
@ItsANameToo hmm.. short answer: I wasn't thinking clearly, long: I was trying to make the compilation works and while trying some alternatives come up with that as a solution to handle the tailwind variables.

I really just needed to add the files that contained the variables used on  to the markdown, [fixed on this last commit](https://github.com/ArkEcosystem/laravel-ui/pull/556/commits/6e74c44f22d5530a473e5de9af107ec0c2796379)